mmsoap/client.py

```diff
@@ -142,27 +144,194 @@ def raise_for_response(self, response):
             pass
 
     def check_replies(self, maximum_replies=None):
```

Comment:
Im not sure about that because it changes the existing behaviour a bit. The ruby sdk also uses None for the default.